sdk/metricsadvisor/Azure.AI.MetricsAdvisor/src/Models/DataFeed/DataFeed.cs

```diff
@@ -21,6 +22,7 @@ namespace Azure.AI.MetricsAdvisor.Models
     /// <see cref="MetricsAdvisorAdministrationClient.CreateDataFeedAsync"/>.
     /// </remarks>
     public class DataFeed
+#pragma warning restore CS0419 // Ambiguous reference in cref attribute
```

Comment:
Is this an ambiguous reference between protocol and convenience methods?  This feels like something we should resolve.

Reply:
Yes, `MetricsAdvisorAdministrationClient.CreateDataFeedAsync` is ambiguous reference between protocol and convenience methods.
Protocol Method 
```C#
public virtual async Task<Response> CreateDataFeedAsync(RequestContent content, RequestContext context = null)
```

Convenience Method 
```C#
public virtual async Task<Response<DataFeed>> CreateDataFeedAsync(DataFeed dataFeed, CancellationToken cancellationToken = default)
```

How do you think we should resolve it in the doc comment?

Reply:
Use `<see cref="MetricsAdvisorAdministrationClient.CreateDataFeedAsync(DataFeed, CancellationToken)"/>` instead, since the model is referring to the convenience API and not the protocol method.